### PR TITLE
PSMDB-1487 Updated Readme and Contrib guides for 8.0

### PR DESCRIPTION
**Pull request by nastena1606**
modified:   CONTRIBUTING.rst
	modified:   README.md

### CODE REVIEW

File: CONTRIBUTING.rst
Comment (ktrushin) [inline CONTRIBUTING.rst:1]: From what I can see, the `Build steps` section has significant amount of duplication. The section has two subsections: `Debian/Ubuntu` and `Red Hat Enterprise Linux and derivatives`. Each of the subsections follows the same structure:
- Install Python and Python modules
- Install operating system dependencies
- Build AWS Software Development Kit for C++ library
- Build Percona Server for MongoDB

However, only the `Install operating system dependencies` item depends on the distribution, be it `Debian/Ubuntu` or `Red Hat Enterprise Linux and derivatives`. All the other items have the exactly same text in `Debian/Ubuntu` and `Red Hat Enterprise Linux and derivatives`. For instance, `Build AWS Software Development Kit for C++ library` in `Debian/Ubuntu` is exactly the same as in `Red Hat Enterprise Linux and derivatives`.

Not only does that duplication make the whole document longer, it also complicates the maintenance. If something needs to be changed in `Build Percona Server for MongoDB`, there is a risk of changing in one place and forgetting about the other. The risk is even higher if another distributions are added in the future, e.g. SuSe.

Duplication also can form incorrect expectations in readers. If the guide distinguishes `Build AWS Software Development Kit for C++ library` in RedHat from `Build AWS Software Development Kit for C++ library` in Debian, then it is natural assume that AWS library for RedHat should be built in another way than for Debian. But in reality, the build steps are the same. The same reasoning applies to `Install Python and Python modules` and `Build Percona Server for MongoDB`. It was the developers' effort to make the build procedure (almost) independent on the distribution by employing virtual environment, cmake, scons, etc. 

Finally, I would reorder the items so that they go from the low_level/most_generic to high_level/most_specific. For instance, Python and Python modules are only required for building Percona Server for MongoDB but the respective item goes first as if building the AWS library depended on them.

To sum up, I suggest the following structure:
- Build steps
  - Install operating system dependencies
    -  Debian/Ubuntu
    - Red Hat Enterprise Linux and derivatives
  - Build AWS Software Development Kit for C++ library
  - Install Python and Python modules
  - Build Percona Server for MongoDB

**Review by igorsol — APPROVED** [review]
LGTM
